pkg/bpf/bpffs.go

```diff
@@ -129,6 +175,35 @@ func mountFS() error {
 		if err != nil {
 			return fmt.Errorf("Command execution failed: %s\n%s", err, out)
 		}
+	} else { // Already mounted. We need to fail if mounted multiple times.
+
+		// Execute the following command to find multiple bpffs mount points
+		// % mount | grep "<mapRoot> " | wc -l | cut -f1 -d' '
+		newmapRoot := mapRoot + " " // Append space to ignore /bpf/xdp and /bpf/ip mountpoints.
+		mt := exec.Command("mount")
+		grep := exec.Command("grep", newmapRoot)
+		wc := exec.Command("wc", "-l")
+		cut := exec.Command("cut", "-f1", "-d ")
```

Comment:
I think you could define this as:
```
cmds := []*exec.Command{
    exec.Command("mount"),
    ...
}
output, stderr, _ := mountCmdPipe(cmds...)
```

Reply:
What if someone did a prior `mount --bind /sys/fs/bpf /sys/fs/bpf` (or some other alternative path cilium should use)? Then we can still mount bpf fs safely, but we would bail out instead here, right?

Reply:
This does not take care of --bind. What would you suggest? @aanm

Reply:
If you want to use shell, one could find out for example this way to check for mount points with `stat` tool, where the bind mount still points to sysfs (or whatever other location) instead of to itself. Starting out from a plain dir:
```
[root@linux-2 ~]# umount /sys/fs/bpf
umount: /sys/fs/bpf: not mounted
[root@linux-2 ~]# stat -c %m /sys/fs/bpf
/sys
[root@linux-2 ~]# mount --bind /sys/fs/bpf /sys/fs/bpf
[root@linux-2 ~]# stat -c %m /sys/fs/bpf
/sys
[root@linux-2 ~]# mount -t bpf bpf /sys/fs/bpf
[root@linux-2 ~]# stat -c %m /sys/fs/bpf
/sys/fs/bpf
```

Reply:
@tgraf : Addressed the review comment